### PR TITLE
Replace ice shard and remove unused ice textures

### LINKED ISSUE #1270: Damaging ice chunks and the particles look the same
which might be a bit confusing

### PR DESCRIPTION
**Pull request by HexapodPhilosopher**
**Brief Description of What This PR Does**

Replaces ice shard model with a snowflake-like ice crystal with a new texture and removes unused ice textures.

**Related Issues**
Closes #1270

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [X] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by CI09** [comment]
You can put `Closes #1270` to make this PR close that issue 👍

File: simulation_parameters/microbe_stage/biomes.json [hunk 1]
```diff
@@ -1861,7 +1861,7 @@
           "Radius": 1,
           "ChunkScale": 1,
           "PhysicsDensity": 4000,
-          "Size": 1000,
+          "Size": 3,
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/biomes.json:1864]: Why did you change the size? I think the size is 1000 here to prevent this being accidentally engulfed (if the engulfed property is accidentally enabled on this).

File: assets/textures/Iceshards.png.import
Comment (hhyyrylainen) [inline assets/textures/Iceshards.png.import:1]: Why was this renamed to iceshard**s**? Isn't this still a single object so this should follow the convention fo the file name not being plural?

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
Tested in-game and everything seemed fine, I just have 2 technical side comments.

File: simulation_parameters/microbe_stage/biomes.json (hunk 1 above)
Comment (hhyyrylainen) [reply simulation_parameters/microbe_stage/biomes.json:1864]: And I'll note that this "size" is really the engulfing size for engulfing calculations and impacts nothing else.

Comment (HexapodPhilosopher) [reply simulation_parameters/microbe_stage/biomes.json:1864]: Ah, okay. I thought the engulfing was intentional. It damages you, but then quickly dissolves. I can change it.

Comment (hhyyrylainen) [reply simulation_parameters/microbe_stage/biomes.json:1864]: The size was 1000 before as it was for gameplay purposes to prevent eating the ice. It could be changed but I'd want someone from the game design team to weigh-in on such a change (I personally think that preventing the player from eating the ice for no other purpose than to receive damage is better than allowing engulfing the ice).

File: assets/textures/Iceshards.png.import
Comment (HexapodPhilosopher) [reply assets/textures/Iceshards.png.import:1]: I was considering adding a second, noticeably different one. Again, I can change the name.

Comment (hhyyrylainen) [reply assets/textures/Iceshards.png.import:1]: Even in that case I'd expect the texture name to be singular. Only if the actual texture image contained multiple things, like for example like a particle texture might do, then I'd think it would be reason enough to break from the file naming convention and have an "unexpected" name for the file.

**Review by hhyyrylainen — APPROVED** [review]
I'll approve this without re-testing as I'll assume based on reading the diff that there likely wasn't any small mistake that slipped in the file rename.
